float64 val # value of the grasp -99 highest -20 minimum
geometry_msgs/Point gcp # grasp point

# points to compute the closing direction of the gripper
geometry_msgs/Point gp1 
geometry_msgs/Point gp2

geometry_msgs/Vector3 approach_vector # approaching vector of the grasp point
float64 roll 
